src/components/multi-select/multi-select-item.ts

```diff
@@ -0,0 +1,98 @@
+import settings from 'carbon-components/es/globals/js/settings';
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { html, property, customElement, LitElement } from 'lit-element';
+import styles from './multi-select.scss';
+
+const { prefix } = settings;
+
+/**
+ * Multi select item.
+ */
+@customElement(`${prefix}-multi-select-item` as any)
```

Comment:
This name is kinda long (not that there's a reasonable alternative) ... but more specifically, I wonder if we could reuse `bx-dropdown-item` at all for multi-select ...